modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/Model.scala

```diff
@@ -139,6 +141,121 @@ object Model {
     */
   case class SequencesQueue[T](queue: List[T])
 
+  // Ported from OCS' SPSiteQuality.java
+
+  final case class Conditions(
+    cc: CloudCover,
+    iq: ImageQuality,
+    sb: SkyBackground,
+    wv: WaterVapor
+  )
+
+  object Conditions {
+
+    val worst = Conditions(
+      CloudCover.Any,
+      ImageQuality.Any,
+      SkyBackground.Any,
+      WaterVapor.Any
+    )
+
+    val nominal = Conditions(
+      CloudCover.Percent50,
+      ImageQuality.Percent70,
+      SkyBackground.Percent50,
+      WaterVapor.Any
+    )
+
+    val best = Conditions(
+      // In the ODB model it's 20% but that value it's marked as obsolete
+      // so I took the non-obsolete lowest value.
+      CloudCover.Percent50,
+      ImageQuality.Percent20,
+      SkyBackground.Percent20,
+      WaterVapor.Percent20
+    )
+
+    implicit val equalConditions: Equal[Conditions] = Equal.equalA[Conditions]
+
+    implicit val showConditions: Show[Conditions] = Show.shows[Conditions] {
+      case Conditions(cc, iq, sb, wv) =>
+        List(cc.shows, iq.shows, sb.shows, wv.shows).intercalate(", ")
```

Comment:
The thing about about `intercalate` is that it works for everything that is a `Monoid` including `String`, but in this case `mkString` seems a better choice.